docs/cogsvc.md

```diff
@@ -0,0 +1,47 @@
+# Azure Cognitive Services on Apache Spark™
+
+## Azure Cognitive Services
+[Azure Cognitive Services](https://azure.microsoft.com/en-us/services/cognitive-services/) are a suite of APIs, SDKs, and services available to help developers build intelligent applications without having direct AI or data science skills or knowledge by enabling developers to easily add cognitive features into their applications. The goal of Azure Cognitive Services is to help developers create applications that can see, hear, speak, understand, and even begin to reason. The catalog of services within Azure Cognitive Services can be categorized into five main pillars - Vision, Speech, Language, Web Search, and Decision.
```

Comment:
Thanks for writing this up! One minor note you might want to add is that HTTP-on-Spark supports calling out to any HTTP endpoint and Cognitive Services-based transformers are one example of using it to build higher-level wrappers. You could just as easily build similar wrappers on top of AWS Rekognition or Lex for example.